gateway-service-impl/src/main/java/org/hypertrace/gateway/service/entity/update/EdsEntityUpdater.java

```diff
@@ -134,4 +137,22 @@ private EntityUpdateRequest convertToEqsUpdateRequest(UpdateEntityRequest update
 
     return eqsUpdateRequestBuilder.build();
   }
+
+  private Map<String, AttributeMetadata> getAttributeMetadataByResultName(
+      UpdateEntityRequest updateRequest, UpdateExecutionContext updateExecutionContext) {
+    return updateRequest.getSelectionList().stream()
```

Comment:
Looks like `AttributeMetadataUtil.remapAttributeMetadataByResultKey` can be reused here?

```
AttributeMetadataUtil.remapAttributeMetadataByResultKey(updateRequest.getSelectionList(), updateExecutionContext.getAttributeMetadata())
```

Reply:
Thanks! I had implemented this multiple times before I realized it was identical and moved it to the shared util, must have missed fixing this one up.